paas_prometheus_exporter/output.tf

```
@@ -1,3 +1,6 @@
 output endpoint {
   value = cloudfoundry_route.paas_prometheus_exporter.endpoint
 }
+output name {
+  value = cloudfoundry_app.paas_prometheus_exporter.name
+}
```

```suggestion
```suggestion
output exporter {
  value = {
   name       = cloudfoundry_route.paas_prometheus_exporter.name
   endpoint = cloudfoundry_app.paas_prometheus_exporter.endpoint
  }
}
```
```